plugins/catalog-react/report-alpha.api.md

```diff
@@ -74,6 +74,23 @@ export const catalogReactTranslationRef: TranslationRef<
   }
 >;
 
+// @alpha (undocumented)
+export type ContextMenuItemComponent = (
+  props: ContextMenuItemProps,
+) => React_2.JSX.Element;
+
+// @alpha (undocumented)
+export const contextMenuItemComponentDataRef: ConfigurableExtensionDataRef<
+  ContextMenuItemComponent,
+  'catalog.contextMenuItemComponent',
+  {}
+>;
+
+// @alpha (undocumented)
+export type ContextMenuItemProps = {
+  onClose: () => void;
```

Comment:
Hmm, this one is a bit awkward, it'd really be something like `onItemClick` or such right? and then that triggers the menu to close. I wonder if we could perhaps handle this with a React context instead? So that we can make these plain React elements with the core data ref instead. The context would be private and consumed by the item wrapper in the blueprint. It'd need to use our singleton utility as there might be duplicates of `catalog-react` installed, see for example https://github.com/backstage/backstage/blob/65454876fb21ec1a8cda1dc556aa1f2af3d874fd/plugins/catalog-react/src/hooks/useEntity.tsx#L36-L38